src/main/java/software/amazon/smithy/lsp/SmithyLanguageServer.java

```diff
@@ -78,11 +78,11 @@ public CompletableFuture<InitializeResult> initialize(InitializeParams params) {
     if (params.getWorkspaceFolders() == null) {
       try {
         tempWorkspaceRoot = Files.createTempDirectory("smithy-lsp-workspace").toFile();
-        System.out.println("Created temporary workspace root: " + tempWorkspaceRoot);
+        LspLog.println("Created temporary workspace root: " + tempWorkspaceRoot);
         tempWorkspaceRoot.deleteOnExit();
         WorkspaceFolder workspaceFolder = new WorkspaceFolder(tempWorkspaceRoot.toURI().toString());
         params.setWorkspaceFolders(ListUtils.of(workspaceFolder));
-      } catch (IOException e) {
+      } catch (Exception e) {
```

Comment:
This catches IOException because the `Files.createTempDirectory` method throws a checked IOException. I don't think we should catch more here, a different error means there's something else wrong and we shouldn't try to recover from that.

Reply:
fair enough, happy to revert this change